Tests/Form/Type/GroupFormTypeTest.php

```diff
@@ -3,24 +3,49 @@
 namespace FOS\UserBundle\Tests\Form\Type;
 
 use FOS\UserBundle\Form\Type\GroupFormType;
+use FOS\UserBundle\Form\Type\RolesFormType;
 use FOS\UserBundle\Tests\TestGroup;
 use FOS\UserBundle\Util\LegacyFormHelper;
+use FOS\UserBundle\Util\RolesHelper;
+use Symfony\Component\Form\PreloadedExtension;
 
 class GroupFormTypeTest extends TypeTestCase
 {
+    private $rolesHelper;
+
+    protected function setUp()
+    {
+        $this->rolesHelper = new RolesHelper(array());
+
+        parent::setUp();
+    }
+
+    protected function getTypeExtensions()
+    {
+        $extensions = parent::getTypeExtensions();
+
+        $type = new RolesFormType($this->rolesHelper);
+
+        return array_merge($extensions, array(
+            new PreloadedExtension(array($type), array()),
```

Comment:
you issue is that ``PreloadedExtension`` is not a form type extension. It is a type extension. This extension is meant to be returned by ``getExtensions``, not by ``getTypeExtensions``